MeetupPlatformApi/Controllers/MeetupsController.cs

```diff
@@ -1,98 +1,77 @@
 ﻿using AutoMapper;
-using MeetupPlatformApi.DataBase;
-using MeetupPlatformApi.DataBase.Entities;
 using MeetupPlatformApi.DataTransferObjects;
+using MeetupPlatformApi.Context;
+using MeetupPlatformApi.Entities;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.EntityFrameworkCore;
 
 namespace MeetupPlatformApi.Controllers;
 
-[Route("api/meetups/")]
+[Route("/api/meetups")]
 [ApiController]
 public class MeetupsController : ControllerBase
 {
-    private readonly MeetupsPlatformContext _meetupsPlatformContext;
+    private readonly ApplicationContext context;
+    private readonly IMapper mapper;
 
-    private readonly IMapper _mapper;
-
-    public MeetupsController(
-        MeetupsPlatformContext meetupsPlatformContext,
-        IMapper mapper)
+    public MeetupsController(ApplicationContext context, IMapper mapper)
     {
-        _mapper = mapper;
-        _meetupsPlatformContext = meetupsPlatformContext;
+        this.context = context;
+        this.mapper = mapper;
     }
 
     [HttpGet]
     public async Task<IActionResult> GetAllMeetups()
     {
-        var meetupsEntity = await _meetupsPlatformContext.Meetups.ToListAsync();
-
-        var meetupsOutputDto = _mapper.Map<List<MeetupOutputDto>>(meetupsEntity);
-
-        return Ok(meetupsOutputDto);
+        var meetups = await context.Meetups.ToListAsync();
+        var outputDtos = mapper.Map<IEnumerable<MeetupOutputDto>>(meetups);
+        return Ok(outputDtos);
     }
 
-    [HttpGet("{id}")]
-    public async Task<IActionResult> GetMeetupById(int id)
+    [HttpGet("{id:int}")]
```

Comment:
Using typed URL templates enables us to:
-   Restrict the provided values set
-   Have separate URLs for different types of values (e.g. get user by a numeric id + get user by username)